src/index.js

```diff
@@ -1,38 +1,119 @@
 import './style.css';
+import './input.css';
+import {
+  addTodo, removeTodo, markCompleted, editDescription, clearCompleted,
+} from './todoCrud.js';
 
-const tasks = [
-  {
-    description: 'Pay the bills',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Pick up the kids',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Write some code',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Drink coffee',
-    completed: true,
-    index: 3,
-  },
-];
-
-const loadTask = (arr) => {
-  arr.forEach((task) => {
-    const container = document.querySelector('.list-div');
-    const listItem = document.createElement('li');
-    listItem.innerHTML = `
-    <input type="checkbox" name="" id="">
-    <p class="task-to-be-done">${task.description}</p>
-    `;
-    container.appendChild(listItem);
-  });
+const todoList = document.querySelector('#todo-list');
+const todoForm = document.querySelector('#todo-form');
+const input = document.querySelector('#todo-input');
+const clearBtn = document.querySelector('#clear-btn');
+
+const showTrash = (index) => {
+  const trash = document.querySelector('.fa-trash');
+  const ellipsis = document.querySelector('.fa-ellipsis-v');
+  const todo = document.getElementById(index);
+  todo.classList.add('bg-orange-100');
+  ellipsis.classList.add('hidden');
+  trash.classList.remove('hidden');
 };
 
-loadTask(tasks);
\ No newline at end of file
+class Todos {
+  constructor() {
+    this.todos = [];
+  }
+
+  add(todo) {
+    this.todos.push(todo);
+  }
+
+  remove(todo) {
+    this.todos = this.todos.filter((t) => t !== todo);
+    this.todos.forEach((t, i) => {
+      t.index = i;
+    });
+  }
+
+  save() {
+    localStorage.setItem('todos', JSON.stringify(this.todos));
+  }
+
```

Comment:
- [ ] when trying to run the project locally as described in the readme file, the JS functionality is not working. in the console the user receives the below error:
![afbeelding](https://github.com/Dorcas126/To-Do-List/assets/39240820/4828bb7e-a4a6-4666-aab6-86075ab380fc)